bin/upload-video

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+# This class processes and uploads a video to s3,
+# making it available for viewing through Code.org's fallback video player.
+
+# Usage:
+# upload-video [youtube id] [video filename]
+#
+# Dependency requirements:
+#
+# ffmpeg (`brew install ffmpeg` / `apt-get install ffmpeg`) - for transcoding manually-provided video files
+# s3_access_key_id and s3_secret_access_key defined (e.g. in `locals.yml`)
+
+# To use this script to process all YouTube video tags in custom levels, run this command from the root folder (between =begin and =end):
```

Comment:
Do you mean checking in this commented-out command as a standalone shell script, so it could be invoked directly from the command-line as opposed to copying/pasting it to the shell as-needed?

My original thought was that it made sense to bury it in a comment here, as I didn't intend for it to be part of any regular process (it's a one-off script), while at the same time preserving it if it's ever needed again in the future. I don't have strong preference on this though so I can commit it as a separate shell script if that's your preference